### PR TITLE
Move from IvoryCKEditorBundle to FOSCKEditorBundle

### PR DESCRIPTION
**Pull request by kunicmarko20**
Hi,

We took over [IvoryCKEditorBundle](https://github.com/egeloen/IvoryCKEditorBundle) and now it is under [FriendsOfSymfony](https://github.com/FriendsOfSymfony) wing and it is renamed to [FOSCKEditorBundle](https://github.com/FriendsOfSymfony/FOSCKEditorBundle).

We are helping active libraries that are depending on `IvoryCKEditorBundle` migrate to `FOSCKEditorBundle`.

The Bundle will be released in the next few days, do not merge until the bundle gets released so we can change the version in the composer.

Thank you for your time.

### CODE REVIEW

**Comment by semin-lev** [comment]
Okay, thank you. Will merge after the bundle will be released.

File: composer.json [hunk 1]
```diff
@@ -13,7 +13,7 @@
     ],
     "require": {
         "a2lix/translation-form-bundle":      "^2.1",
-        "egeloen/ckeditor-bundle":            "^4.0",
+        "friendsofsymfony/ckeditor-bundle": "dev-master",
```
Comment (kunicmarko20) [inline composer.json]: I didn't change this to "^1.0" do you want me to open a new PR or you will do it?

Comment (kunicmarko20) [reply composer.json]: cc @semin-lev  @IgorNikolaev

Comment (IgorNikolaev) [reply composer.json]: We'll do it, thank you.

Comment (semin-lev) [reply composer.json]: @kunicmarko20 if it's not hard for you, please, open new PR. Thank you.

Comment (semin-lev) [reply composer.json]: @kunicmarko20 we did, thank you.

**Pull request merged**